vulnfeeds/cves/versions.go

```diff
@@ -256,3 +272,29 @@ func CPEs(cve CVEItem) []string {
 	}
 	return cpes
 }
+
+// Parse a well-formed CPE string into a struct.
+func ParseCPE(cpe_str string) (*CPE, error) {
+	if !strings.HasPrefix(cpe_str, "cpe:") {
+		return nil, errors.New("Invalid CPE string")
```

Comment:
nit: define a global error `ErrInvalidCPE` and reuse here. 

See https://pkg.go.dev/net/http#pkg-variables for similar examples.

Reply:
Actually I think using an error is overkill. I'll take a leaf out of [go/gotip/episodes/86](http://go/gotip/episodes/86) instead.